docs/api/conformance/Metadata-Tests.md

```diff
@@ -16,54 +16,54 @@ You will need to pre-populate the AppDirectory with the following items:
 
 ## Retrieve `AppMetadata`
 
-| App | Step           | Details                                                                                           |
-|-----|----------------|---------------------------------------------------------------------------------------------------|
-| Test   | 1.getAppMetadata    | Retrieve metadata for the configured app A with <br /> `const metadata1 = await fdc3.getAppMetadata({appId: "<A's appId>"})`  |
-| Test   | 2.Confirm    | Compare the `AppMetadata` object to the expected definition for the fields provided above during setup and ensure that the metadata matches. An `instanceId` should NOT be set  |
+| App    | Step              | Details                                                                                           |
+|--------|-------------------|---------------------------------------------------------------------------------------------------|
+| Test   | 1. getAppMetadata | Retrieve metadata for the configured app A with <br/> `const metadata1 = await fdc3.getAppMetadata({appId:"<A's-appId>"})`  |
```

Comment:
is this `-` needed in `{appId:"<A's-appId>"}`?

Reply:
I don't remember why I added the hyphens - it was possibly to stop it breaking lines?